tests/lib/lore_test.sh

```diff
@@ -170,4 +195,102 @@ function test_extract_metadata_from_patch_title_rfc()
   assertEquals "($LINENO)" "$output" "$expected"
 }
 
+function test_create_lore_bookmarked_file()
+{
+  rm "${BOOKMARKED_SERIES_PATH}"
```

Comment:
Make sure `"${BOOKMARKED_SERIES_PATH}"` is the path that you want to remove before calling `rm`.